roles/core/time/defaults/main.yml

```diff
@@ -0,0 +1,24 @@
+---
+# Those defaults values should be overwriten by your inventory
+# Start the services by default
+start_services: true
+enable_services: true
+# Default NTP configuration
+external_time:
+  time_server:
+    server:
+      - 0.fr.pool.ntp.org
+      - 1.fr.pool.ntp.org
+      - 2.fr.pool.ntp.org
+      - 3.fr.pool.ntp.org
+    pool:
+      - pool.ntp.org
+  time_client:
+    server: []
+    pool: []
+time_zone: Europe/Paris
+
+# Default server configuration
+time_profile: 'server'
+icebergs_system: false
+network_interfaces: []
```

Comment:
Same than for DNS, defaults are not allowed for this kind of variables.

Reply:
ok, then I will move them in the converge.yml as pre_tasks, ok?

Reply:
yes